objects/9ad187/script.lua

```diff
@@ -588,7 +590,7 @@ function difficultyCheck(params)
     if optionalExtraBoard then
         local intNum = math.floor(difficulty / 3) + 2
         difficulty = difficulty + intNum
-        if alternateSetupIndex == 2 then
+        if boardLayout == "Thematic" then
```

Comment:
just noticed there's a bug here (unrelated to your change), but could you also add a `or param.thematic` here?

Reply:
actually nvm, i'll just fix it real quick, noticed a couple other things too